src/utils/is.js

```diff
@@ -56,3 +56,20 @@ export const isAbortSignal = object => {
 		)
 	);
 };
+
+/**
+ * isDomainOrSubdomain reports whether sub is a subdomain (or exact match) of
+ * the parent domain.
+ *
+ * Both domains must already be in canonical form.
+ * @param {string|URL} original
+ * @param {string|URL} destination
+ */
+export const isDomainOrSubdomain = (destination, original) => {
```

Comment:
The names of the variables here don't see to match the values passed in?

```
if (!isDomainOrSubdomain(request.url, locationURL)) {
```

Surely locationURL is the destination and request.url is the original?

Reply:
I think you are right. it works as intended and how it is suppose to protect you, But the variable names are mixed up

Reply:
No rush to get fixed, can create a issue about it if needed